### PR TITLE
add profiles to doc

### PR DESCRIPTION
**Pull request by mmulholla**
Long overdue docs update on profiles.

### CODE REVIEW

File: docs/helm-chart-checks.md [hunk 1]
```diff
@@ -128,6 +128,39 @@ This section provides help on the basic usage of Helm chart checks with the podm
...
+  - The ```helm-lint``` check is mandatory with all other checks optional.  
+- default
+  - Profile used if a specific one is not specified.
+  - All checks are mandatory.
+
+Each profile also has a version and currently all profiles are v1.0.
+
+To specify which profile to use the --set flag:
+```
+    --set profile.vendorType=partner
```
Comment (dperaza4dustbit) [inline docs/helm-chart-checks.md:151]: Is this case sensitive?

**Review by dperaza4dustbit — COMMENTED** [review]
One question

File: docs/helm-chart-checks.md [hunk 2]
```diff
@@ -128,6 +128,39 @@ This section provides help on the basic usage of Helm chart checks with the podm
...
+  - All checks are mandatory.
+- redhat 
+  - Defines the requirements for a red hat internal chart to pass helm chart certfication.
+  - All checks are mandatory.
+- community
+  - Defines the requirements for a community chart to pass helm chart certfication.
+  - The ```helm-lint``` check is mandatory with all other checks optional.  
+- default
+  - Profile used if a specific one is not specified.
+  - All checks are mandatory.
```
[the remark this answers is not in the document]
Comment (mmulholla) [reply docs/helm-chart-checks.md:145]: Done.

File: docs/helm-chart-checks.md (hunk 1 above)
Comment (mmulholla) [reply docs/helm-chart-checks.md:151]: No - added a note to this effect.

**Comment by mmulholla** [comment]
Closing this as now covered by https://github.com/redhat-certification/chart-verifier/pull/216

**Pull request closed without merging**